tests/Backend/Workspaces/ParallelWorkspacesTestCase.php

```diff
@@ -0,0 +1,124 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: marc
+ * Date: 04/07/2016
+ * Time: 10:52
+ */
+
+namespace Keboola\Test\Backend\Workspaces;
+
+use Keboola\Db\Import\Snowflake\Connection;
+use Keboola\StorageApi\Client;
+use Keboola\StorageApi\Options\TokenCreateOptions;
+use Keboola\StorageApi\Workspaces;
+use Keboola\Test\StorageApiTestCase;
+
+abstract class ParallelWorkspacesTestCase extends StorageApiTestCase
+{
+    /** @var string */
+    protected $runId;
+
+    /** @var Client */
+    protected $workspaceSapiClient;
+
+    public function setUp()
+    {
+        parent::setUp();
+        $this->initEmptyTestBucketsForParallelTests();
+
+        $description = get_class($this) . '\\' . $this->getName();
+
+        $this->deleteOldTestTokensAndWorkspaces($description);
```

Comment:
Tadyto ještě asi upravím. Aby se ten token mazal/zakládal znovu jen pokud by neměl nastavený `canManageBuckets`. Ať se to zbytečně nezakládá s každým testem.